index.html

```diff
@@ -16,10 +16,133 @@
+        <input type="text" name="company" placeholder="Company">
+      </form>
+      <img id="profile-image" src="/img/profile-image-placeholder.svg" alt="profile image placeholder" />
+    </header>
+    <main>
+      <form class="user-input">
+        <img class="add-button" src="/img/add-button.svg" alt="plus sign" />
+        <h3>Phone:</h3>
+        <input type="text" name="phone number" placeholder="primary">
+        <input type="text" name="phone number" placeholder="secondary">
+      </form>
+      <form class="user-input">
+        <img class="add-button" src="/img/add-button.svg" alt="plus sign" />
+        <h3>Email:</h3>
+        <input type="text" name="email" placeholder="primary">
+        <input type="text" name="email" placeholder="secondary">
+      </form>
+      <form class="user-input">
+        <img class="add-button" src="/img/add-button.svg" alt="plus sign" />
+        <h3>Social:</h3>
+        <input type="text" name="social-media" placeholder="primary">
+      </form>
+    </main>
+    <footer id="home-screen-footer">
+      <button><img class="footer-buttons" src="img/back-button.svg" alt="back button" /></button>
+      <button><img class="footer-buttons" src="img/add-contact-button.svg" alt="add contact button" /></button>
+      <button><img class="footer-buttons" src="img/contact-list-button.svg" alt="contact list button" /></button>
+      <button><img class="footer-buttons" src="img/exchange-button.svg" alt="exchange info button" /></button>
+    </footer>
+  </section>
+  <section id="social-media-layover">
+    <header class="profile-header">
+      <form>
+        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="company" placeholder="Company">
+      </form>
+      <img id="profile-image" src="/img/profile-image-placeholder.svg" alt="profile image placeholder" />
+    </header>
+    <main id="social-media-icons">
+      <img class="social-icons" src="img/facebook-icon.svg" alt="facebook icon" />
+      <img class="social-icons" src="img/twitter-icon.svg" alt="facebook icon" />
+      <img class="social-icons" src="img/linkedin-icon.svg" alt="facebook icon" />
+      <img class="social-icons" src="img/instagram-icon.svg" alt="facebook icon" />
+      <img class="social-icons" src="img/google-icon.svg" alt="facebook icon" />
+      <img class="social-icons" src="img/skype-icon.svg" alt="facebook icon" />
+    </main>
+  </section>
+  <section id="add-new-contact">
+    <h3>Add New Contact</h3>
+    <header class="profile-header">
+      <form>
+        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="company" placeholder="Company">
+      </form>
+      <img id="profile-image" src="/img/profile-image-placeholder.svg" alt="profile image placeholder" />
+    </header>
+    <main>
+      <form class="user-input">
+        <img class="add-button" src="/img/add-button.svg" alt="plus sign" />
```

Comment:
Yes that makes sense. Sometimes I get confused with the alt tags because I want them to be descriptive of what the image is, but yes that can be changed.